test/e2e_node/gubernator.sh

```diff
@@ -22,11 +22,12 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
-source hack/lib/logging.sh
+KUBE_ROOT=$(dirname "${BASH_SOURCE[0]}")/../..
+source "${KUBE_ROOT}/hack/lib/logging.sh"
 
 
 if [[ $# -eq 0 || ! $1 =~ ^[Yy]$ ]]; then
-  read -p "Do you want to run gubernator.sh and upload logs publicly to GCS? [y/n]" yn
+  read -r -p "Do you want to run gubernator.sh and upload logs publicly to GCS? [y/n]" yn
```

Comment:
For my own knowledge - why is `-r` necessary here?

Reply:
without `-r` read will mangle escape sequences. _technically_ it shouldn't matter in this context, but it's a good habit and a correctness fix unless you actually wanted that behavior.

Reply:
the shellcheck wiki has a good entry on this https://github.com/koalaman/shellcheck/wiki/SC2162